packages/devtools_app/lib/src/memory/flutter/memory_chart.dart

```diff
@@ -71,9 +69,13 @@ class MemoryChartState extends State<MemoryChart> with AutoDisposeMixin {
   void didChangeDependencies() {
     super.didChangeDependencies();
 
+    final newController = Controllers.of(context).memory;
+    if (newController == controller) return;
+    controller = newController;
```

Comment:
What is the purpose of this code? Curious because other screens that initialize the controller in `didChangeDependencies` don't have this check

Reply:
Was mimicking  e.g., TimelineScreenBodyState, _FlutterFramesChartState, etc.  However, it doesn't appear I need the check just:

`controller = Controllers.of(context).memory;`